src/js/ripple/log.js

```diff
@@ -149,7 +149,11 @@ var NullLogEngine = {
 if (typeof window !== 'undefined' && typeof console !== 'undefined') {
   Log.engine = InteractiveLogEngine;
 } else if (typeof console !== 'undefined' && console.log) {
-  Log.engine = BasicLogEngine;
+  if (process.env.NODE_ENV === 'test') {
+    Log.engine = NullLogEngine;
```

Comment:
My original comment on this got lost: "This seems dangerous. What log messages are you trying to hide?"